docs/dev/news/4843.fix

```diff
@@ -0,0 +1 @@
+Python compiler emits reserved names for datatypes
```

Comment:
```suggestion
The Python compiler no longer emits reserved names for datatypes
```
?

Reply:
Okay sure. It hasn't been our pattern, but since it ends up under a "Bug fixes" header in the release notes this style works too.